ur_msgs/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_message_files(
 add_service_files(
    FILES
    SetPayload.srv
+   SetSpeedSlider.srv
```

Comment:
> So, `SetSpeedSliderFraction` would probably be technically correct. Should we go for that?

If it sets the fraction, then yes.

Otherwise the current name would be OK.

Reply:
Thanks @gavanderhoorn I think, this discussion is then resolved.